Files/MultilingualResources/Files.hi-IN.xlf

```diff
@@ -1948,6 +1948,22 @@
           <source>Enable individual preferences for individual directories</source>
           <target state="new">Enable individual preferences for individual directories</target>
         </trans-unit>
+        <trans-unit id="AddPaneTeachingTip.Subtitle" translate="yes" xml:space="preserve">
+          <source>Click to open the right pane</source>
+          <target state="new">Click to open the right pane</target>
+        </trans-unit>
+        <trans-unit id="AddPaneTeachingTip.Title" translate="yes" xml:space="preserve">
```

Comment:
I don't think we need this teaching tip since the feature will be for powerusers and most users won't find it very helpful.

Reply:
I don't consider dual pane view as poweruser feature

Reply:
I suggest we add more teaching tips AROUND the dual pane view

Reply:
@d2dyno1 Perhaps that's because you are a poweruser 😄
The design of an app should be usable by the majority of users without having to explain how everything works. If you find the need to explain how a feature works, then most of the time, you need to rethink the feature. Teaching tips are meant to be used when there is no other way for a user to learn how to use a feature however, sometimes, some features need a guide or more information and that is where the wiki can fill the gap.

Reply:
I added so the user will know he/she has to hover near the right edge of the window to popup the "add new pane" button. Otherwise it may not be obvious

Reply:
Adding on to my comment about the `AlwaysOpenDualPaneInNewTab` setting, I think that it won't be as necessary to inform the user of this method since there will be a right click option as well.